whats-on/terraform/services.tf

```diff
@@ -0,0 +1,34 @@
+module "whats_on" {
+  source             = "git::https://github.com/wellcometrust/terraform.git//services?ref=v1.0.4"
```

Comment:
If we use the `eventbrite` service as an example, we use the new version of the modules (v3)

Reply:
As discussed, will keep this consistent with the event and exhibitions services and then we can look at updating them all together